README.md

```diff
@@ -1,194 +1,195 @@
-
-- NuGet packages for [Blob][], [File][], [Queue][]
-- [Azure Storage APIs for .NET][]
-- Quickstart for [Blob][blob-quickstart], [File][file-quickstart], [Queue][queue-quickstart]
-
-## Target Frameworks
-
-- .NET Framework 4.5.2: As of September 2018, Storage Client Libraries for .NET supports primarily the desktop .NET Framework 4.5.2 release and above.
-- Netstandard1.3: Storage Client Libraries for .NET are available to support Netstandard application development including Xamarin/UWP applications. 
-- Netstandard2.0: Storage Client Libraries for .NET are available to support Netstandard2.0 application development including Xamarin/UWP applications. 
-
-## Requirements
-
-- Microsoft Azure Subscription: To call Microsoft Azure services, you need to first [create an account][]. Sign up for a free trial or use your MSDN subscriber benefits.
-- Hosting: To host your .NET code in Microsoft Azure, you additionally need to download the full Microsoft Azure SDK for .NET - which includes packaging,
-    emulation, and deployment tools, or use Microsoft Azure Web Sites to deploy ASP.NET web applications.
-
-## Versioning Information
-
-- The Storage Client Libraries use [the semantic versioning scheme][semver]
-
-## Use with the Azure Storage Emulator
-
-- The Client Libraries use a particular Storage Service version. In order to use the Storage Client Libraries with the Storage Emulator, a corresponding minimum version of the Azure Storage Emulator must be used. Older versions of the Storage Emulator do not have the necessary code to successfully respond to new requests.
-- Currently, the minimum version of the Azure Storage Emulator needed for this library is 5.3. If you encounter a `VersionNotSupportedByEmulator` (400 Bad Request) error, please [update the Storage Emulator.][emulator]
-
-## Download & Install
-
-The Storage Client Libraries ship with the Microsoft Azure SDK for .NET and also on NuGet. You'll find the latest version and hotfixes on NuGet via the `Microsoft.Azure.Storage.Blob`, `Microsoft.Azure.Storage.File`, `Microsoft.Azure.Storage.Queue`, and `Microsoft.Azure.Storage.Common` packages. 
-
-### Via Git
-
-To get the source code of the SDK via git just type:
-
-```bash
-git clone git://github.com/Azure/azure-storage-net.git
-cd azure-storage-net
-```
-
-### Via NuGet
-
-To get the binaries of this library as distributed by Microsoft, ready for use
-within your project you can also have them installed by the .NET package manager: [Blob][], [File][], [Queue][].
-
-Please note that the minimum NuGet client version requirement has been updated to 2.12 in order to support multiple .NET Standard targets in the NuGet package.
-
+> As of 9.4.0, the Table service is not supported by this library.
+> Table support is being provided by [CosmosDB][Microsoft.Azure.Cosmos.Table].

```

Comment:
Nuget link is missing :)

Reply:
https://www.nuget.org/packages/Microsoft.Azure.Cosmos.Table

Reply:
It wasn't missing.  It is a feature of markdown to create a set of named links and reference them.  See for example lines 9 and 76, and the items starting at line 159.